src/app/board/question/all/page.tsx

```
@@ -80,9 +81,32 @@ export default function AllQuestionPage() {
+    setCurrentPage(0);
+    await fetchAllQuestions(0, defaultFiltering);
+  };
+
+  const handleRemoveFilter = async (filterType: "sortType" | "chaetaekStatus" | "tag", value?: string) => {
+    const newFiltering: Partial<QuestionCommand> = { ...currentFiltering };
+
+    if (filterType === "sortType") {
+      delete newFiltering.sortType;
+    } else if (filterType === "chaetaekStatus") {
+      delete newFiltering.chaetaekStatus;
+    } else if (filterType === "tag" && value) {
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)